connectivity/FEATURE_BLE/source/cordio/source/GattServerImpl.cpp

```diff
@@ -810,6 +872,9 @@ ble_error_t GattServer::areUpdatesEnabled(
     bool *enabled
 )
 {
+    tr_info("Determine if updates are enabled in characteristic %s",
```

Comment:
Please add the trace before the return so you can print the result. This is a getter so I would use a `tr_debug` instead of a `tr_info`. You can provide context in the `tr_error`.